tests/cmd/check-spelling/kata-spell-check.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright (c) 2019 Intel Corporation
+# Copyright (c) 2019-2023 Intel Corporation
```

Comment:
@jodh-intel Do we need to update these at all? As far as I understand, these should be changed when a new version is published.